### ISSUE TITLE
Single-letter shortcuts in interactive rebase view 

### ISSUE THREAD

**Issue by dandavison** [kahole/vscode-magit#131]
Minor magit compatibility request:

The `git-rebase-todo`  view is the view in which one prepares an interactive rebase (example below). It would be nice to be able to use single-letter shortcuts, as one can in magit. For example, with the cursor at beginning-of-line, at the start of the  word `pick`, I would like to be able to type `f` and have `pick`  change to `fixup`, etc.

```
pick 656a1f9 Upgrade to GitHub-native Dependabot (#579)
pick 0e99a10 Fix ansi syntax theme (#581)

# Rebase 0de020a..0e99a10 onto 0de020a (2 commands)
#
# Commands:
# p, pick <commit> = use commit
# r, reword <commit> = use commit, but edit the commit message
# e, edit <commit> = use commit, but stop for amending
# s, squash <commit> = use commit, but meld into previous commit
# f, fixup <commit> = like "squash", but discard this commit's log message
# x, exec <command> = run command (the rest of the line) using shell
# b, break = stop here (continue rebase later with 'git rebase --continue')
# d, drop <commit> = remove commit
# l, label <label> = label current HEAD with a name
# t, reset <label> = reset HEAD to a label
# m, merge [-C <commit> | -c <commit>] <label> [# <oneline>]
```

**Comment by praveenperera** [comment]
This would be awesome, +1

**Comment by jacobemcken** [comment]
@kahole this seems like a very popular request... any thoughts on it?

**Comment by kahole** [comment]
Yeah, it's a reasonable request given how it works in magit. Personally, I really like using normal text editing to edit the rebasing buffer and find it more natural.
I definitely understand people being more efficient with the single-click commands though, and with the popularity of this issue I'm definitely open to having those single-click keybindings be the default.

This is also where this project gets (is) awkward, trying to balance the expectations of how magit and vscode work. At some points edamagit has strayed from magit when a vscode style implementation was deemed just as good. And other times vscode usability/expectations has been thrown out the window in favor of magit behaviour, e.g. the single-key menu system.

**Comment by grgar** [comment]
There's a couple of extensions that do this now

- [Git Rebase Shortcuts](https://marketplace.visualstudio.com/items?itemName=trentrand.git-rebase-shortcuts), popular and always active regardless of vim mode, and
- [Git Rebase Keybindings](https://marketplace.visualstudio.com/items?itemName=grg.rebase), made by me[^1], designed for vim/edamagit but missing functionality (e.g. no multi-line editing)

[^1]: I'm not just here shilling my extension, I've got 3 PRs open here
